TreeSearch.java

```diff
@@ -0,0 +1,82 @@
+import java.util.Scanner;
+import java.util.LinkedList;
+import java.util.Queue;
+
+public class TreeSearch {
+
+	Node root;
+
+	public void TreeQueue(Node root) { //queue traversal
```

Comment:
the name TreeQueue is hinting that this is a data structure, but here it is a method. So next time just call it something like BFS